requirements.txt

```diff
@@ -3,3 +3,5 @@ pydantic>=2.4,<2.5
 uvicorn>=0.24,<0.25
 aiohttp>=3.8,<3.9
 vcfpy>=0.13.8
```

Comment:
Variation team uses specific tag on GitHub. @jonk-ebi Was suppose to make this change in the earlier PR.

Reply:
This PR doesn't have anything to do with vcfpy, We should keep an eye on it when reviewing Jyo's PR.

Reply:
Ah yes, true